tests/rundb/test_sqldb.py

```diff
@@ -154,61 +159,27 @@ def test_list_tags(db: SQLDB, db_session: Session):
 
 
 def test_projects(db: SQLDB, db_session: Session):
-    prj1 = {
-        "name": "p1",
-        "description": "banana",
-        # 'users': ['u1', 'u2'],
-        "spec": {"company": "ACME"},
-        "state": "active",
-        "created": datetime.now(),
-    }
-    pid1 = db.add_project(db_session, prj1)
-    p1 = db.get_project(db_session, project_id=pid1)
-    assert p1, f"project {pid1} not found"
-    out = {
-        "name": p1.name,
-        "description": p1.description,
-        # 'users': sorted(u.name for u in p1.users),
-        "spec": p1.spec,
-        "state": p1.state,
-        "created": p1.created,
-    }
-    assert prj1 == out, "bad project"
-
-    data = {"description": "lemon"}
-    db.update_project(db_session, p1.name, data)
-    p1 = db.get_project(db_session, project_id=pid1)
-    assert data["description"] == p1.description, "bad update"
-
-    prj2 = {"name": "p2"}
-    db.add_project(db_session, prj2)
-    prjs = {p.name for p in db.list_projects(db_session)}
-    assert {prj1["name"], prj2["name"]} == prjs, "list"
-
-
-def test_cache_projects(db: SQLDB, db_session: Session):
-    assert 0 == len(db._projects), "empty cache"
-    name = "prj348"
-    db.add_project(db_session, {"name": name})
-    assert {name} == db._projects, "project"
-
-    mock = Mock()
-    with patch(db, add_project=mock):
-        db._ensure_project(db_session, name)
-    mock.assert_not_called()
-
-    mock = Mock()
-    with patch(db, add_project=mock):
-        db._ensure_project(db_session, name + "-new")
-    mock.assert_called_once()
-
-    project_2_name = "project-2"
-    db.add_project(db_session, {"name": project_2_name})
-    db._projects = set()
-    mock = Mock()
-    with patch(db, add_project=mock):
-        db._ensure_project(db_session, name)
-    mock.assert_not_called()
+    project = mlrun.api.schemas.Project(
+        name="p1", description="banana", spec={"other_field": "value"}, state="active"
+    )
+    db.create_project(db_session, project)
+    project_output = db.get_project(db_session, name=project.name)
```

Comment:
having to do get after each create / patch / store (on all levels) to get the result is really repeating itself and adding a lot of lines of code in several modules

Reply:
This is testing the deepest module - the DB - which IMO shouldn't return the created/updated/patched object

Reply:
Answered to that in other places